SDK lacks hexdump

Sanity checks for `secure-kernel-loader` require `hexdump` during build, but this command is not available in SDK:

```shell
trenchboot@7376a7e71262:~/secure-kernel-loader$ make
cc -D__ASSEMBLY__ -m64 -Os -g -MMD -MP -march=btver2 -mno-sse -mno-mmx -fpie -fomit-frame-pointer -Iinclude -ffreestanding -fno-common -Wall -Werror -c head.S -o head.o
cc -m64 -Os -g -MMD -MP -march=btver2 -mno-sse -mno-mmx -fpie -fomit-frame-pointer -Iinclude -ffreestanding -fno-common -Wall -Werror -o dev.o -c dev.c
cc -m64 -Os -g -MMD -MP -march=btver2 -mno-sse -mno-mmx -fpie -fomit-frame-pointer -Iinclude -ffreestanding -fno-common -Wall -Werror -o event_log.o -c event_log.c
cc -m64 -Os -g -MMD -MP -march=btver2 -mno-sse -mno-mmx -fpie -fomit-frame-pointer -Iinclude -ffreestanding -fno-common -Wall -Werror -o iommu.o -c iommu.c
cc -m64 -Os -g -MMD -MP -march=btver2 -mno-sse -mno-mmx -fpie -fomit-frame-pointer -Iinclude -ffreestanding -fno-common -Wall -Werror -o main.o -c main.c
cc -m64 -Os -g -MMD -MP -march=btver2 -mno-sse -mno-mmx -fpie -fomit-frame-pointer -Iinclude -ffreestanding -fno-common -Wall -Werror -o pci.o -c pci.c
cc -m64 -Os -g -MMD -MP -march=btver2 -mno-sse -mno-mmx -fpie -fomit-frame-pointer -Iinclude -ffreestanding -fno-common -Wall -Werror -o printk.o -c printk.c
cc -m64 -Os -g -MMD -MP -march=btver2 -mno-sse -mno-mmx -fpie -fomit-frame-pointer -Iinclude -ffreestanding -fno-common -Wall -Werror -o sha1sum.o -c sha1sum.c
cc -m64 -Os -g -MMD -MP -march=btver2 -mno-sse -mno-mmx -fpie -fomit-frame-pointer -Iinclude -ffreestanding -fno-common -Wall -Werror -o sha256.o -c sha256.c
cc -m64 -Os -g -MMD -MP -march=btver2 -mno-sse -mno-mmx -fpie -fomit-frame-pointer -Iinclude -ffreestanding -fno-common -Wall -Werror -o string.o -c string.c
cc -m64 -Os -g -MMD -MP -march=btver2 -mno-sse -mno-mmx -fpie -fomit-frame-pointer -Iinclude -ffreestanding -fno-common -Wall -Werror -include boot.h -include errno-base.h -include byteswap.h -DEBADRQC=EINVAL -o tpmlib/crb.o -c tpmlib/crb.c
cc -m64 -Os -g -MMD -MP -march=btver2 -mno-sse -mno-mmx -fpie -fomit-frame-pointer -Iinclude -ffreestanding -fno-common -Wall -Werror -include boot.h -include errno-base.h -include byteswap.h -DEBADRQC=EINVAL -o tpmlib/tis.o -c tpmlib/tis.c
cc -m64 -Os -g -MMD -MP -march=btver2 -mno-sse -mno-mmx -fpie -fomit-frame-pointer -Iinclude -ffreestanding -fno-common -Wall -Werror -include boot.h -include errno-base.h -include byteswap.h -DEBADRQC=EINVAL -o tpmlib/tpm.o -c tpmlib/tpm.c
cc -m64 -Os -g -MMD -MP -march=btver2 -mno-sse -mno-mmx -fpie -fomit-frame-pointer -Iinclude -ffreestanding -fno-common -Wall -Werror -include boot.h -include errno-base.h -include byteswap.h -DEBADRQC=EINVAL -o tpmlib/tpm1_cmds.o -c tpmlib/tpm1_cmds.c
cc -m64 -Os -g -MMD -MP -march=btver2 -mno-sse -mno-mmx -fpie -fomit-frame-pointer -Iinclude -ffreestanding -fno-common -Wall -Werror -include boot.h -include errno-base.h -include byteswap.h -DEBADRQC=EINVAL -o tpmlib/tpm2_auth.o -c tpmlib/tpm2_auth.c
cc -m64 -Os -g -MMD -MP -march=btver2 -mno-sse -mno-mmx -fpie -fomit-frame-pointer -Iinclude -ffreestanding -fno-common -Wall -Werror -include boot.h -include errno-base.h -include byteswap.h -DEBADRQC=EINVAL -o tpmlib/tpm2_cmds.o -c tpmlib/tpm2_cmds.c
cc -m64 -Os -g -MMD -MP -march=btver2 -mno-sse -mno-mmx -fpie -fomit-frame-pointer -Iinclude -ffreestanding -fno-common -Wall -Werror -include boot.h -include errno-base.h -include byteswap.h -DEBADRQC=EINVAL -o tpmlib/tpm_buff.o -c tpmlib/tpm_buff.c
cc -m64 -Os -g -MMD -MP -march=btver2 -mno-sse -mno-mmx -fpie -fomit-frame-pointer -Iinclude -ffreestanding -fno-common -Wall -Werror -include boot.h -include errno-base.h -include byteswap.h -DEBADRQC=EINVAL -o tpmlib/tpmio.o -c tpmlib/tpmio.c
cc -Wl,-T,link.lds -m64 -nostdlib -no-pie -Wl,--build-id=none head.o dev.o event_log.o iommu.o main.o pci.o printk.o sha1sum.o sha256.o string.o tpmlib/crb.o tpmlib/tis.o tpmlib/tpm.o tpmlib/tpm1_cmds.o tpmlib/tpm2_auth.o tpmlib/tpm2_cmds.o tpmlib/tpm_buff.o tpmlib/tpmio.o -o skl
objcopy -O binary -S skl skl.bin
util.sh: line 3: hexdump: command not found
./sanity_check.sh: line 4: hexdump: command not found
od: invalid --skip-bytes argument ''
ERROR: SKL UUID missing or misplaced in skl.bin
make: *** [Makefile:55: skl.bin] Error 1
```

--- a/Dockerfile
+++ b/Dockerfile
@@ -16,6 +16,7 @@ RUN \
 		bin86 \
 		bison \
 		bridge-utils \
+		bsdextrautils \
 		build-essential \
 		bzip2 \
 		ccache \

